src/h5web/providers/utils.ts

```diff
@@ -75,6 +75,14 @@ export function assertGroup(entity: HDF5Entity): asserts entity is HDF5Group {
   }
 }
 
+export function assertSimpleShape(
+  shape: HDF5Shape
+): asserts shape is HDF5SimpleShape {
+  if (shape.class !== HDF5ShapeClass.Simple) {
+    throw new Error('Expected simple shape');
+  }
+}
```

Comment:
Used in `getMockDatasetDims`. I thought it best to place it here, next to some of the other assertion function.